gum/arch-arm64/gumarm64writer.c

```diff
@@ -1992,15 +1993,37 @@ gum_arm64_writer_commit_literals (GumArm64Writer * self)
     if (r->width != GUM_LITERAL_64BIT)
       continue;
 
+    /*
+     * Whilst instructions in aarch64 are always in little endian (even on
```

Comment:
```suggestion
     * Whilst instructions in AArch64 are always in little endian (even on
```

(Here and elsewhere.) For consistency with our existing comments.

Reply:
Done